processor/src/main/java/org/mapstruct/ap/internal/model/CollectionAssignmentBuilder.java

```diff
@@ -0,0 +1,163 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
```

Comment:
Think the builder is a good idea, But should we express somehow in the name of the class that this belongs to the property mapping ?